api/auth/auth.go

```diff
@@ -200,27 +200,25 @@ func (auth *Auth) WrapHandler(h http.Handler) http.Handler {
 		auth.lock.RUnlock()
 
 		if err != nil { // Probably because signature wrong
-			w.WriteHeader(http.StatusUnauthorized)
 			// Error is intentionally dropped here as there is nothing left to
 			// do with it.
-			_, _ = io.WriteString(w, fmt.Sprintf("invalid auth token: %s", err))
-			return
-		}
-		if !token.Valid { // Check that token isn't expired
```

Comment:
Removed this `if block` as it is sort of a dead code. The jwt package sets token as valid if and only if there is no validation error. We already cover the case that it returns an error in [here](https://github.com/ava-labs/avalanchego/pull/714/files#diff-26263389709bb74db688fd6e5917683aecf93bbe964d8196895e7f32f0986447L202) . You can check the jwt package code [here](https://github.com/dgrijalva/jwt-go/blob/dc14462fd58732591c7fa58cc8496d6824316a82/parser.go#L82)